packages/react/src/app-layout/AppLayoutSidebarDialog.tsx

```diff
@@ -154,19 +160,19 @@ function CloseMenuButton({
 			alignItems="center"
 			borderBottom
 			borderColor="muted"
-			paddingLeft={1}
 			flexShrink={0}
+			paddingLeft={1}
 		>
 			<Flex
-				as={BaseButton}
-				onClick={onClick}
-				flexDirection="column"
 				alignItems="center"
-				gap={0.5}
+				as={BaseButton}
 				color="action"
+				flexDirection="column"
```

Comment:
`Stack` - sorry, I do see these aren't your code and that you've just moved things around.

Reply:
Yeah, that's why I'm starting to dislike this "alphabetise as we go" approach. PRs look more complicated and require more scrutiny than they should be getting